pkg/collector/collector.go

```diff
@@ -61,6 +61,12 @@ func (p *PluginNotFoundError) Error() string {
 	return fmt.Sprintf("no plugin found for %s", p.metricTypeName)
 }
 
+func (p *PluginNotFoundError) Is(target error) bool {
+	t, ok := target.(*PluginNotFoundError)
+	_ = t
```

Comment:
```suggestion
	_, ok := target.(*PluginNotFoundError)
```

quick :nail_care:, the rest it looks good to me.